package.json

```diff
@@ -61,6 +61,7 @@
     "karma-coverage-istanbul-reporter": "^1.2.1",
     "karma-jasmine": "~1.1.0",
     "karma-jasmine-html-reporter": "^0.2.2",
+    "node-sass": "^4.13.1",
```

Comment:
Please revert this change as this is not related to this PR.

Reply:
Sir as i mentioned above the build is failing without this so i need to add it

Reply:
https://travis-ci.org/Cloud-CV/EvalAI-ngx/builds/640847824?utm_source=github_status&utm_medium=notification

Reply:
I have removed it lets see if build is successful

Reply:
@Sanji515 Sir the build is failing without it. so should i add it?

Reply:
@Sanji515 Sir should i add it?